### PR TITLE
Node header

### PR DESCRIPTION
**Pull request by KeeyanGhoreshi**
removes back button, makes your mouse a pointer on node headers.

### CODE REVIEW

**Comment by dehall** [comment]
So the back button change looks good. Tested with adding `onBack={() => window.history.back()}` to the Navigation item in App.tsx and it worked as expected.
About the node header pointer cursor, it's also a pointer on the Explore Pathways screen where it's not actually clickable. It should probably only appear clickable on the main pathways view. Maybe with a new className that only gets set on the Node if the onClickHandler isn't null.
Last thing is this needs a rebase.

Also I deleted the Jira issue for this when you said the entire node title was already clickable. I forgot about the pointer cursor thing. I'll add that back now.

**Comment by danlee1025** [comment]
Can this branch get a rebase?  I'll look at it then.

**Comment by KeeyanGhoreshi** [comment]
rebased

**Comment by danlee1025** [comment]
This looks pretty good, once the cursor pointer is removed in the explore pathways screen I'll approve.

File: src/styles/_nodes.scss [hunk 1]
```diff
@@ -29,6 +29,10 @@
   justify-content: flex-end;
 }
 
+.clickable {
+    cursor: pointer;
```
Comment (dehall) [inline src/styles/_nodes.scss]:
```suggestion
  cursor: pointer;
```
Code style is this should be indented 2 spaces

**Review by dehall — APPROVED** [review]

**Review by radamson — APPROVED** [review]

**Pull request merged**